proto/task_service.proto

```diff
@@ -35,6 +35,7 @@ message CreateTaskRequest {
   batch_plan.TaskId task_id = 1;
   batch_plan.PlanFragment plan = 2;
   uint64 epoch = 3;
+  common.VNodeRanges vnode_ranges = 4;
```

Comment:
Why `vnode_ranges` is an attribute of task? I think it's the attribute of table scan plan node.

Reply:
BTW, would it be more general to make it a `VNode` list?

Reply:
> Why vnode_ranges is an attribute of task? I think it's the attribute of table scan plan node.

Because it's the result of scheduling - Each scan task scans different ranges. Putting it in scan plan node is also possible, but requires rewriting each task's plan node. Both ok to me.

> BTW, would it be more general to make it a VNode list?

Previously it's vnode bitmap, but I think list/bitmap may be difficult for scan executor to use?

Reply:
We're introducing yet another representation! 🤣